lib/Frontend/ParseableInterfaceModuleLoader.cpp

```diff
@@ -760,16 +861,18 @@ class ParseableInterfaceModuleLoaderImpl {
 
   // Check that the output .swiftmodule file is at least as new as all the
   // dependencies it read when it was built last time.
-  bool serializedASTBufferIsUpToDate(
+  bool serializedASTBufferIsUpToDate(StringRef path,
```

Comment:
Nitpick: please don't split parameters like this. Either they wrap to a double-indent because one of them is too long, or they align to the open-paren.